client/homebrew/editor/editor.jsx

```diff
@@ -74,6 +76,10 @@ const Editor = createClass({
 
 	componentDidUpdate : function(prevProps, prevState, snapshot) {
 		this.highlightCustomMarkdown();
+		if(this.props.jumpSource) {
+			this.sourceJump();
+			this.setState({sourceJump: false});
+		}
```

Comment:
Is this bit of code actually doing anything? Does `this.props.jumpSource` ever get set? And does `this.state.sourceJump` ever get used?